PackageInfo.g

```diff
@@ -68,26 +71,48 @@ Persons := [
     Place         := "Aachen",
     Institution   := "Lehrstuhl D für Mathematik, RWTH Aachen",
   ),
+rec(
+    LastName      := "Hähndel",
+    FirstNames    := "Paula",
+    IsAuthor      := false,
+    IsMaintainer  := false,
+    Email         := "paula.haehndel@mathematik.uni-halle.de",
+    WWWHome       := "https://algebra.mathematik.uni-halle.de/haehndel/",
+    Place         := "Halle (Saale)",
+    Institution   := "Martin Luther University of Halle-Wittenberg"
+  ),
+  rec(
+    LastName      := "Hulpke",
+    FirstNames    := "Alexander",
+    IsAuthor      := false,
+    IsMaintainer  := false,
+    Email         := "hulpke@colostate.edu",
+    WWWHome       := "https://www.math.colostate.edu/~hulpke/",
+    Place         := "Fort Collins",
+    Institution   := "Colorado State University"
+  ),
   rec(
     LastName      := "Howe",
     FirstNames    := "Stephen",
     IsAuthor      := false,
     IsMaintainer  := false,
-    PostalAddress := "Unknown",
+  ),
+  rec(
+    LastName      := "Jefferson",
+    FirstNames    := "Christopher",
+    IsAuthor      := false,
+    IsMaintainer  := false,
+    Email         := "caj21@st-andrews.ac.uk",
+    WWWHome       := "https://caj.host.cs.st-andrews.ac.uk",
+    Place         := "St. Andrews",
```

Comment:
Sorry to poke you again about this, but I'd like it if you removed the full stop after the 'St' in this line and the next.

Reply:
All good, it was my mistake, sorry for making you point this out to me again. I've fixed it.